public/js/components/RightSidebar.js

```diff
@@ -88,15 +88,15 @@ const RightSidebar = React.createClass({
     keyShortcuts.on("F8", this.resume);
     keyShortcuts.on("F10", this.stepOver);
     keyShortcuts.on("F11", this.stepIn);
-    keyShortcuts.on("F12", this.stepOut);
+    keyShortcuts.on("Shift+F11", this.stepOut);
   },
 
   componentWillUnmount() {
     const { keyShortcuts } = this.props;
     keyShortcuts.off("F8", this.resume);
     keyShortcuts.off("F10", this.stepOver);
     keyShortcuts.off("F11", this.stepIn);
-    keyShortcuts.off("F12", this.stepOut);
+    keyShortcuts.off("Shift+F11", this.stepOut);
```

Comment:
Why this change? (may have miss the conversation) Is this what the old debugger does? It'd be good to keep the same shortcuts if possible, but there's a good reason that's ok.